UI/frontend-plugins/decklink-output-ui/CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+project(decklink-output-ui)
+
+if (APPLE)
+	find_library(COCOA Cocoa)
+	include_directories(${COCOA})
+endif ()
+
+if (UNIX AND NOT APPLE)
+	find_package(X11 REQUIRED)
+	link_libraries(${X11_LIBRARIES})
+	include_directories(${X11_INCLUDE_DIR})
+endif ()
+
+set(decklink-ouput-ui_HEADERS
+		${decklink-ouput-ui_HEADERS}
```

Comment:
Style:  Tab once instead of twice to indent.  Think of the contents of `set` as enclosed rather than continued lines.